.gitignore

```diff
@@ -1,407 +1,5 @@
-# Dropbox settings and caches
```

Comment:
Probably rewriting `.gitignore` deserves its own PR?

Reply:
sure I will create another PR with this

Reply:
removed from PR